tests/tasks/cleanup_candlepin_container.yml

```
@@ -0,0 +1,37 @@
+# SPDX-License-Identifier: MIT
+---
+- name: Check if the candlepin container is started
+  command: podman container exists candlepin
```

please use the `argv:` syntax for `command`, ie
```suggestion
  command:
    argv:
      - podman
      - container
      - exists
      - candlepin
```
While it is more verbose, it is consistent with what already used everywhere in the role, and it has the big advantage of not requiring shell split by the `command` module.